src/mac.js

```diff
@@ -237,6 +251,9 @@ class MacApp extends App {
 
     if (this.usageDescription) {
       for (const [type, description] of Object.entries(this.usageDescription)) {
+        for (const plistKey of plistsToUpdate) {
+          this[plistKey][`NS${type}UsageDescription`] = description
+        }
```

Comment:
_nit_ `NS${type}UsageDescription` is built length+1 times; could be moved outside of the inner loop and built just once